cli/parser/bazelrc/bazelrc.go

```diff
@@ -58,6 +59,18 @@ var (
 		"coverage": "test",
 	}
 
+	commonPhases = map[string]struct{}{
+		"common": {},
+		"always": {},
+	}
```

Comment:
The behavior of these two flags is different enough that I feel like it's kind of confusing to have them in the same map. I would maybe use two separate top-level constants for these.

Reply:
I made top-level constants, but since the map is used specifically for the function below and isn't exported, I just renamed the map to match the name of the renamed function.